examples/observability/eks-java-jmx/variables.tf

```diff
@@ -0,0 +1,8 @@
+# variable "grafana_workspace_id" {}
+variable "grafana_endpoint" {}
+
+variable "grafana_api_key" {
+  type        = string
+  sensitive   = true
+  description = "Api key for authorizing the Grafana provider to make changes to Amazon Managed Grafana"
+}
```

Comment:
You can just these two variables as expected and remove the `tfvars` file to further simplify